src/libs/actions/Policy/Policy.ts

```diff
@@ -2722,6 +2723,58 @@ function enableExpensifyCard(policyID: string, enabled: boolean) {
     }
 }
 
+function enableCompanyCards(policyID: string, enabled: boolean) {
+    const authToken = NetworkStore.getAuthToken();
+    if (!authToken) {
+        return;
+    }
```

Comment:
Why do we need this?

Reply:
Probably because authToken can be 
`function getAuthToken(): string | null | undefined {
    return authToken;
}`
Not see any reason to fire API call if token is not exists - API will failed

Reply:
Why/how could we have no authToken? 
Also, I don't see us adding this check to all our API calls and almost all of them would fail without an authToken